README.md

```diff
@@ -102,6 +102,10 @@ The following variable defaults are defined in `defaults/main.yml`.
 
     list of IP addresses to ban (default: `[]`)
 
+* `minecraft_server_properties`
+
+    list of server.properties entries (e.g. `server-port=25565`) to set (default: `[]`)
```

Comment:
Hm I'm not sure how to do this in a generic way. The argument spec for any ServerFile is currently server_file (string), path (string), and values (list). I guess I could add in another option, say, values_dict or something, and pass both those to the ServerFile constructor, but that doesn't seem to simplify anything. And as you say, sharing code between modules seems pretty much impossible, so I'd have to duplicate a lot to have a separate module that takes 'values' as a list.

Is there a better way, do you think?